scenarios/load_tests/load_with_different_stakes.yml

```diff
@@ -0,0 +1,28 @@
+# This scenario runs a load test with different stakes for the validator nodes.
+# it generates a constant load throughout the test duration, it should be possible
+# to observe the effect of different stakes on submission of transactions.
+name: load with different stakes
+duration: 120
+
+# Initial validator nodes in the network.
+validators:
+  - name: large
+    stake: 5000000000
```

Comment:
Do validators even have enough tokens for that? As commented above, they should just have 10M tokens each.